src/borg/helpers.py

```diff
@@ -1585,6 +1622,7 @@ def get_item_data(self, archive):
             'archive': remove_surrogates(archive.name),
             'id': bin_to_hex(archive.id),
             'time': format_time(to_localtime(archive.ts)),
+            'start': format_time(to_localtime(archive.ts)),
```

Comment:
wondering about why it needs same value as in "time".
also, if there is start, where is end?